dashboard/app/controllers/application_controller.rb

```diff
@@ -144,6 +147,13 @@ def milestone_response(options)
       response[:share_failure] = options[:share_failure]
     end
 
+    if HintViewRequest.enabled?
```

Comment:
it looks like before it was inside of `if current_user` and this isn't? Was that intentional?